easybuild/easyconfigs/o/OpenMPI/OpenMPI-1.6.3-iccifort-2013.3.163.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenMPI'
+version = '1.6.3'
+
+homepage = 'http://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-2 implementation."""
+
+toolchain = {'name': 'iccifort', 'version': '2013.3.163'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.open-mpi.org/software/ompi/v%(version_major_minor)s/downloads']
+
+configopts = '--with-threads=posix --enable-shared --enable-mpi-thread-multiple --with-openib '
+configopts += '--enable-mpirun-prefix-by-default '  # suppress failure modes in relation to mpirun path
+configopts += '--with-hwloc=$EBROOTHWLOC --with-pmi '  # hwloc support
```

Comment:
@boegel: Hi Kenneth! Sorry for not getting back to you earlier! I haven't found any reference online (in the OpenMPI website or elsewhere) about any problems caused on other system when building OpenMPI with PMI. Unfortunately this is something we cannot test here since all our systems are running SLURM at the moment.